curl_cffi/requests/cookies.py

```diff
@@ -290,6 +290,20 @@ def delete(
         for cookie in remove:
             self.jar.clear(cookie.domain, cookie.path, cookie.name)
 
+    def get_dict(self, domain: Optional[str] = None, path: Optional[str] = None) -> dict:
+        """Takes as an argument an optional domain and path and returns a plain
+        old Python dict of name-value pairs of cookies that meet the
+        requirements.
+
+        """
+        dictionary = {}
+        for cookie in self.jar:
+            if (domain is None or cookie.domain == domain) and (
+                    path is None or cookie.path == path
+            ):
+                dictionary[cookie.name] = cookie.value
+        return dictionary
```

Comment:
I don't really understand the purpose here, may be it's expected, but cookie matching is not working like that. You can check official RFC:

* https://datatracker.ietf.org/doc/html/rfc6265#section-5.1.3 * https://datatracker.ietf.org/doc/html/rfc6265#section-5.1.4